vllm/v1/core/kv_cache_utils.py

```diff
@@ -1342,4 +1381,60 @@ def get_kv_cache_configs(
         if len(kv_cache_config.kv_cache_groups) > 0:
             _report_kv_cache_config(vllm_config, kv_cache_config)
 
+    # TODO: Remove before merging. Only for testing.
+    print("kv_cache_configs", kv_cache_configs[0])
+
     return kv_cache_configs
+
+
+class BlockHashListWithBlockSize:
+    """
+    Convert the block hashes under hash_block_size to another target_block_size.
+    Only support scaling up the block size by an integer factor now. Implemented
+    by concatenating the block hashes under hash_block_size to form that of
+    target_block_size.
```

Comment:
maybe we should add the example in the PR description to here? And I think it's worth mention that this is a "lazy" conversion.